demisto_sdk/commands/common/hook_validations/playbook.py

```diff
@@ -62,3 +64,21 @@ def is_no_rolename(self):  # type: () -> bool
             self.is_valid = False
             return False
         return True
+
+    def is_root_connected_to_all_tasks(self):  # type: () -> bool
+        """Check whether the playbook root is connected to all tasks
+
+        Return:
+            bool. if the Playbook has root is connected to all tasks.
+        """
+        start_task_id = self.current_file.get('starttaskid')
+        tasks = self.current_file.get('tasks', {})
+        tasks_bucket = set()
+        next_tasks_bucket = set()
+        for task_id, task in tasks.items():
+            if task_id != start_task_id:
+                tasks_bucket.add(task_id)
+            next_tasks = task.get('nexttasks', {})
+            for next_task_ids in next_tasks.values():
+                next_tasks_bucket.update(next_task_ids)
+        return tasks_bucket.issubset(next_tasks_bucket)
```

Comment:
I think we should add a warning print here. maybe even consider printing the list of task names that ar unreachable from the root.